### PR TITLE
Add holidays for Catalonia

### PR DESCRIPTION
**Pull request by ferranp**
For information, read and make sure you're okay with the [CONTRIBUTING document](https://github.com/novafloss/workalendar/blob/master/CONTRIBUTING.rst#adding-new-calendars).

- [x] Tests with a significant number of years to be tested for your calendar.
- [x] Docstrings for the Calendar class and specific methods.
- [x] Calendar country / label added to the README.rst file,
- [x] Changelog amended with a mention like: "Added ``<country>`` by ``@pseudo`` (#)"

### CODE REVIEW

File: workalendar/tests/test_europe.py [hunk 1]
```diff
@@ -766,7 +766,7 @@ def test_year_2016(self):
         self.assertIn(date(2016, 12, 25), holidays)  # Natal
 
 
-class Spain(GenericCalendarTest):
+class SpainTest(GenericCalendarTest):
```
Comment (brunobord) [inline workalendar/tests/test_europe.py]: nice catch... I don't know how it could work... :/

File: workalendar/tests/test_europe.py [hunk 2]
```diff
@@ -792,6 +792,44 @@ def test_year_2016(self):
...
+        self.assertIn(date(2015, 12, 25), holidays)
+        self.assertIn(date(2015, 12, 26), holidays)
+
+    def test_year_2016(self):
+        holidays = self.cal.holidays_set(2016)
+        self.assertIn(date(2016, 1, 1), holidays)
+        self.assertIn(date(2016, 1, 6), holidays)
+        self.assertIn(date(2016, 3, 25), holidays)
+        self.assertIn(date(2016, 3, 28), holidays)
+        # self.assertIn(date(2016, 5, 16), holidays)
```
Comment (brunobord) [inline workalendar/tests/test_europe.py]: no need to keep this comment

**Comment by brunobord** [comment]
the tests look great; I've checked the checkbox. You'd have to at least edit the README and the Changelog and check all the checkboxes when you're ready.

**Comment by ferranp** [comment]
Ok done!

Changed README and CHANGELOG and removed comented code.

**Comment by brunobord** [comment]
ah. almost done. I've digged the travis build and spotted an issue: https://travis-ci.org/novafloss/workalendar/jobs/148501567 if you look at this, you see there's no test_europe is executed, which is sad, because the coverage report doesn't seem to see it.
I think I know why, you have an ImportError. This import error has been introduced by the *chmod* you committed on various files: `workalendar/europe/__init__.pyworkalendar/europe/__init__.py`, `workalendar/europe/spain.py`, and also especially `workalendar/tests/test_europe.py`. I tried launch the tests from your branch using the previous standard chmod (644) and it worked again. Could you please change the access rights to the given files in order to have all the modules tested accordingly? thx.

**Comment by brunobord** [comment]
for the record, it looks like it's a well-known ~~bug~~feature in nose: http://stackoverflow.com/questions/2378146/why-does-nose-finds-tests-in-files-with-only-644-permission

**Comment by brunobord** [comment]
@ferranp do you need some help?

**Comment by ferranp** [comment]
Ok, file permissoions restored. 
Sorry for this, I work in a mixded Windows/linux environment and sometimes the permissions get changed without noticing.

**Comment by brunobord** [comment]
@ferranp no problem, it happens (and BTW, I'd have never thought it had caused such an issue...)

**Pull request merged**